src/mono/mono/component/debugger-agent.c

```diff
@@ -9002,6 +8994,33 @@ type_commands_internal (int command, MonoClass *klass, MonoDomain *domain, guint
 		buffer_add_byte (buf, m_class_get_rank (klass));
 		break;
 	}
+	case MDBGPROT_CMD_TYPE_GET_FIELD_RVA: 
+	{
+		gpointer iter = NULL;
+		int field_token = decode_int (p, &p, end);
+		while ((f = mono_class_get_fields_internal (klass, &iter))) {
+			if (mono_class_get_field_token (f) == field_token)
+			{
+				if (f->type->attrs & FIELD_ATTRIBUTE_HAS_FIELD_RVA)
+				{
+					int swizzle = 1;
+					int align;
+				#if G_BYTE_ORDER != G_LITTLE_ENDIAN
+					swizzle = mono_type_size (type, &align);
+				#endif
+
+					int dummy;
+					int count = mono_type_size (f->type, &dummy)/mono_type_size (f->type, &align);
+					const char* arr = mono_field_get_rva (f, swizzle);
+					m_dbgprot_buffer_add_byte_array (buf, (uint8_t *)arr, count);
```

Comment:
Can we make a helper function in `object-internals.h`/`object.c` and share it with `ves_icall_System_Runtime_CompilerServices_RuntimeHelpers_GetSpanDataFrom`